01-JS-DOM-APIs/Ex02/index.html

```diff
@@ -0,0 +1,34 @@
+<!DOCTYPE html>
+<html>
+   <head>
+		<meta charset="UTF-8">
```

Comment:
Looks like, is a copy and paste error! Take care every time you use that shortcut, is really useful but It could involve you in trouble.